### PR TITLE
[QA-1912] Fix html handler in app.yaml

### PR DESCRIPTION
**Pull request by aweng98**
Slack [discussion](https://broadinstitute.slack.com/archives/C01EHNUM73R/p1655928477889469?thread_ts=1655916043.984709&cid=C01EHNUM73R).
https://broadworkbench.atlassian.net/browse/QA-1912

We don't want the second handler for the `.html` also handles `index.html` file. Making second handler more specific.

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ handlers:
   http_headers:
     X-Content-Type-Options: "nosniff"
 
-- url: /(.*\.html)
+- url: /(google.*\.html)
   mime_type: text/html
   static_files: build/\1
   upload: build/.*\.html
```
Comment (nawatts) [inline app.yaml]: Should `upload` also be specific to the `google*.html` files?
https://cloud.google.com/appengine/docs/standard/nodejs/config/appref#handlers_element

Comment (aweng98) [reply app.yaml]: Yes. I updated `upload` seconds before saw your comment.

**Review by nawatts — APPROVED** [review]
Tested the following:
* `curl -D - https://pr-9-dot-bvdp-saturn-dev.appspot.com/`
* `curl -D - https://pr-9-dot-bvdp-saturn-dev.appspot.com/index.html`
* `curl -D - https://pr-9-dot-bvdp-saturn-dev.appspot.com/google04dfa0c4e6a35632.html`

**Review by breilly2 — APPROVED** [review]
👍 HTTP headers for `index.html` now look good.
```
$ curl -I https://pr-9-dot-bvdp-saturn-dev.appspot.com
HTTP/2 200
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
x-xss-protection: 1; mode=block
etag: "tmGBtQ"
x-cloud-trace-context: 28ff5d892b2ebaa8f90ff80c2f48904f
content-type: text/html
date: Thu, 23 Jun 2022 13:19:29 GMT
pragma: no-cache
expires: Fri, 01 Jan 1990 00:00:00 GMT
cache-control: no-cache, must-revalidate
server: Google Frontend
content-length: 1720
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
```

**Pull request merged**